ls1intum/Artemis
# `General`: Fix some typos, grammar errors and warnings in the documentation

### Checklist
#### General



### Motivation and Context


Found a typo while writing my thesis in the docker-related parts of the documentation.
Fixed that typo, some warnings and also adopted some changes suggested by Grammarly.

### Description

This PR fixes a typo and adds some grammar fixes suggested by Grammarly.

### Steps for Testing


1. Check the documentation [here](https://artemis-platform--6743.org.readthedocs.build/en/6743/)

### Review Progress






#### Code Review
#### Manual Tests
